app/assets/javascripts/blob/upload_file.js.coffee

```diff
@@ -0,0 +1,35 @@
+Dropzone.options.blobFileDropzone = false;
+
+$(document).on "page:load", ->
+      $("#blobFileDropzone").dropzone()
+
+Dropzone.options.blobFileDropzone = {
+  autoProcessQueue: false,
+  clickable: true,
+  uploadMultiple: false,
+  paramName: "file",
+  parallelUploads: 1,
+  maxFiles: 1,
+  addRemoveLinks: true,
+  previewsContainer: '.dropzone-previews',
+  headers:
+        "X-CSRF-Token": $("meta[name=\"csrf-token\"]").attr("content"),
+  
+  init: -> 
+    submitButton = document.querySelector('#submit-all')
+    blobFileDropzone = this
+    blobFileDropzone.options.maxFilesize = gon.max_file_size or 10
+    submitButton.addEventListener 'click', (e)->
+      e.preventDefault()
+      e.stopPropagation()
+      alert "Please select a file" if blobFileDropzone.getQueuedFiles().length == 0
+      blobFileDropzone.processQueue()
+      return
+    @on 'success', (file, response)->
+      window.location.href = response.filePath
+      return
+    @on 'maxfilesexceeded', (file) ->
+      alert 'Only one file is allowed'
```

Comment:
@stanhu I say you already remove this alert in your forked repository https://github.com/stanhu/gitlabhq/commit/ff3ff0c1748b559771c2a1a39bcd229d2fe12725

I also saw your test for dropzone with text file, I also tested it locally and I think it is working properly, thanks. How should we integrate?